src/test/datascience/notebook/helper.ts

```diff
@@ -192,9 +192,9 @@ export async function canRunNotebookTests() {
     }
     const api = await initialize();
     const appEnv = api.serviceContainer.get<IApplicationEnvironment>(IApplicationEnvironment);
-    const canRunTests = appEnv.extensionChannel !== 'stable';
+    const canRunTests = appEnv.channel === 'insiders';
```

Comment:
We can run notebook tests only in VS Code insiders for now.
Might want to remove this condition when the API stabilizes, else it will fail in stable with main branch.